### DISCUSSION TITLE
Cal-Heatmap and React

### DISCUSSION THREAD

**Discussion by ghost** [wa0x6e/cal-heatmap#339] [Q&A] [answered]
Hello,

I was attempting to use Cal-Heatmap in a project using React and Typescript. I'm having a really hard time getting anything to render using the react template found [here](https://github.com/wa0x6e/cal-heatmap-doc/blob/main/src/components/CalHeatmapComponent/index.tsx). Is there a working example in React outside of the documentation?

**Comment by wa0x6e** [comment]
I will try creating something like https://github.com/wa0x6e/cal-heatmap-vue-starter

**Comment by wa0x6e** [comment, marked as answer]
Repo is live: https://github.com/wa0x6e/cal-heatmap-react-starter
